stacks/spark-trino-superset-s3/hive-metastore.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: hive
 spec:
   image:
-    productVersion: 4.0.1
+    productVersion: 3.1.3 # Cannot upgrade to 4.0 until this is fixed https://github.com/apache/iceberg-python/issues/1222
```

Comment:
Does it work with 4.0.0? I know 2-3 customers with HMS 4.0.0 and iceberg.
If so, we should probably keep that as a supported version

Reply:
This was the hive bump https://github.com/stackabletech/demos/pull/165

Reply:
> Does it work with 4.0.0? 

It does.

Reply:
Then we definitly should keep 4.0.0 as well and mark 4.0.1 as experimental?

Reply:
That sounds like a good plan to me!
Thanks @razvan for catching this!

Reply:
Created a PR to re-add hive 4.0.0 [here](https://github.com/stackabletech/docker-images/pull/1031)

Reply:
That PR is approved now. This one should be good to go.